crates/loader/src/local.rs

```diff
@@ -88,7 +89,7 @@ impl LocalLoader {
 
         let variables = variables
             .into_iter()
-            .map(|(name, v)| Ok((name.to_string(), locked_variable(v)?)))
+            .map(|(name, v)| Self::env_checker((name.to_string(), locked_variable(v)?)))
```

Comment:
Maybe a nit, but doing this in the loader produces an error message that implies we can't understand the manifest (usually meaning invalid TOML or incorrect schema): that's potentially misleading.